docs/usage.md

```diff
@@ -0,0 +1,170 @@
+### Intro to Kubernetes Usage
+
+Once Kubernetes is running and healthy you can interact with it via REST API or using the kubectl command line tool.
+
+- [Configure the Kubernetes Endpoint](#config)
+- [Create a Kubernetes Pod Definition](#pod)
+- [Create a Kubernetes Service Definition](#service)
+- [Launch a Kubernetes Pod via REST API](#api)
+- [Launch a Kubernetes Pod via kubectl](#kubectl)
+- [View Usage Metrics](#metrics)
+
+<a name="config"></a>
+#### Configure the Kubernetes Endpoint
+
+Export the Kubernetes URL to an environment variable, where `<hostname>` is the Mesos Master hostname:
+
+```
+export KUBERNETES_MASTER=http://<hostname>/service/kubernetes/api
+```
+
+<a name="pod"></a>
+#### Create a Kubernetes Pod Definition
+
+A [pod](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/pods.md) is one or more containers that get co-located on the same host. In this example we’re creating a pod with one container, which runs nginx.
+
+1. Download an example nginx pod definition:
+
+    ```
+    mkdir -p examples && curl -o examples/pod-nginx.json https://raw.githubusercontent.com/mesosphere/kubernetes-mesos/master/examples/pod-nginx.json
+    ```
+
+<a name="service"></a>
+#### Create a Kubernetes Service Definition
+
+A [service inside kubernetes](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md) is an abstraction that gives a pod (or an external service) a named endpoint.
+In order to expose the nginx server to the web, you must create a Kubernetes service that refers to it.
+
+1. Download an example nginx service definition:
+
+    ```
+    mkdir -p examples && curl -o examples/service-nginx.json https://raw.githubusercontent.com/mesosphere/kubernetes-mesos/master/examples/service-nginx.json
```

Comment:
dunno. we kind of have two audiences: those using master head and those using a specific version... We don't have dev binaries being built and hosted automatically yet. So we can't just tell them to download everything. Maybe we should just have instructions for master...